charts/pega/charts/installer/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```

Comment:
If this description is put anywhere, we should make this more descriptive for the installer.